cryfs-conf.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -ex
```

Comment:
Automatic configuration and build with Macports-installed libraries that live in /opt/local. Just to spare me the need of providing cmake with sh*tload of "-DXXX -DYYY -DZZZ ...".